Makefile

```
@@ -166,9 +166,11 @@ endif
 export override RUSTFLAGS += $(patsubst %,--cfg %, $(THESEUS_CONFIG))
 
 
-### Convenience targets for building Theseus with all features enabled.
+### Convenience targets for building the entire Theseus workspace
+### with all optional features enabled. 
+### See `theseus_cargo/Cargo.toml` for more details about what this includes.
 all: full
-full : export override CARGOFLAGS += --all-features
+full : export override FEATURES += --workspace --all-features
```

```suggestion
full: FEATURES += --all-features
```
`FEATURES` already contains `--workspace`. You can check this by inspecting the Cargo command printed out when running `make full`.

---

yep, that's true, and i double-checked this. However, I left it in because if someone sets `FEATURES` on the make command line, then it won't contain `--workspace`, and i think it should it someone is running `make full`.

For ex, if you run `make full FEATURES="--feature hello"` then `--workspace` will not be passed into `cargo build`, so some crates will be excluded (the ones not used as dependencies by any of the optional features.

I have also confirmed that it's okay to pass in `--workspace` multiple times, so it's better to have it appear twice than to not have it appear at all.